guides/source/developers/payments/payment-service-providers.html.md

```diff
@@ -64,3 +64,13 @@ You would need to extend or rewrite this class with your preferred PSP
 integration.
 
 [credit-card-base]: https://github.com/solidusio/solidus/blob/master/core/app/models/spree/payment_method/credit_card.rb
+
+### Switching payment service provider
+
+After switching payment service provider, there may be `Spree::PaymentMethod` 
+records referencing a `type` class that does not exist anymore.
+
```

Comment:
I'd specify that this prevents the system to raise the `ActiveRecord::SubclassNotFound` error. This way, it will also be easier to find if someone is searching using the exception name as a keyword.

Reply:
Nice suggestion, I didn't consider it at first. I just added it.